.github/workflows/build-release-binary.yml

```
@@ -89,7 +89,7 @@ jobs:
         run: |
           cd maker-frontend;
           yarn
-          APP=maker yarn build
+          yarn build
```

We could write this as:

```suggestion
          cd ${{ matrix.bin }}-frontend;
          yarn
          yarn build
```

And get rid of the `if`.

---

wouldn't this also make it a bit harder to find out which job failed? (right now they're named separately, so you see at the first glance which frontend failed on the CI)